docs/index.md

```diff
@@ -0,0 +1,49 @@
+# YamlConfiger --- AMQ configurator
+
+This tool can generate a set of configuration files mainly needed for
+AMQ Broker, but it is not limited to only generating files for one product.
+
+YamlConfiger has a user facing Command Line Tool for quick and easy command line usage.
+Furthermore, it is possible to use its API in your python code.
+
+## Documentation
+
+Formatted documentation can be viewed at [rh-messaging-qe.github.io/YamlConfiger/index.html](https://rh-messaging-qe.github.io/YamlConfiger/index.html).
+
+## Contributing
+
+If you find a bug or room for improvement, submit either a ticket or PR.
+
+## Contributors
+
+_Alphabetically ordered_
+
+* Michal Tóth <mtoth@redhat.com>
+* Otavio Piske <opiske@redhat.com>
+* Sean Davey <sdavey@redhat.com>
+* Zdenek Kraus <zkraus@redhat.com> (maintainer)
```

Comment:
```suggestion
## Maintainers

_Alphabetically ordered_

* Michal Tóth <mtoth@redhat.com>
* Dominik Lenoch <dlenoch@redhat.com>
* Zdenek Kraus <zkraus@redhat.com>

See GitHub statistics for an [up-to-date list of contributors](https://github.com/rh-messaging-qe/YamlConfiger/graphs/contributors).
```

Reply:
I don't think I am a maintainer. Probably just Zdenek and Dominik are :)

Reply:
Ok, I'll delete you.